### PR TITLE
Rename Kaia network names

### PR DESCRIPTION
**Pull request by ian0371**
## Proposed changes

- Rename Cypress -> Mainnet, Baobab -> Testnet
- Provides backward compatibility in flags, configuration files

## Types of changes

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: blockchain/genesis.go [hunk 1]
```diff
@@ -168,10 +168,10 @@ func findBlockWithState(db database.DBManager) *types.Block {
 // SetupGenesisBlock writes or updates the genesis block in db.
 // The block that will be used is:
 //
-//	                     genesis == nil                            genesis != nil
+//	                     genesis == nil                             genesis != nil
 //	                  +-------------------------------------------------------------------
-//	db has no genesis |  main-net default, baobab if specified  |  genesis
-//	db has genesis    |  from DB                                |  genesis (if compatible)
+//	db has no genesis |  main-net default, Testnet if specified  |  genesis
```
Comment (yoomee1313) [inline blockchain/genesis.go]:
```suggestion
//	db has no genesis |  main-net default, Testnet if specified  |  genesis
```
```suggestion
//	db has no genesis |  mainnet default, testnet if specified  |  genesis
```

File: cmd/homi/setup/flags.go [hunk 2]
```diff
@@ -41,24 +41,24 @@ var (
...
+	mainnetFlag = &cli.BoolFlag{
+		Name:    "mainnet",
+		Usage:   "Generate genesis.json similar to the one used for Mainnet",
 		Aliases: []string{},
 	}
-	baobabTestFlag = &cli.BoolFlag{
-		Name:    "baobab-test",
-		Usage:   "Generate genesis.json similar to the one used for Baobab with shorter intervals for testing",
+	testnetTestFlag = &cli.BoolFlag{
+		Name:    "testnet-test",
```
Comment (yoomee1313) [inline cmd/homi/setup/flags.go:55]: Can we still accept `baobab-test` here? Maybe some scripts use `baobab-test`.

**Review by blukat29 — DISMISSED** [review]

File: blockchain/genesis.go (hunk 1 above)
Comment (ian0371) [reply blockchain/genesis.go]: fixed a9576d2a

File: cmd/homi/setup/flags.go (hunk 2 above)
Comment (ian0371) [reply cmd/homi/setup/flags.go:55]: fixed a9576d2a

**Review by blukat29 — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Review by JayChoi1736 — APPROVED** [review]

**Pull request merged**